### PR TITLE
fix: allow to generate multiple operators from different reconcilers

### PR DESCRIPTION
**Pull request by Sgitario**
Added unit test to verify this scenario.
Manually tested using the ping pong sample and annotating the two reconcilers with `@CSVMetadata(name = "ping-operator")` and the same with the pong reconciler.

### CODE REVIEW

File: bundle-generator/deployment/src/test/java/io/quarkiverse/operatorsdk/bundle/sources/FirstReconciler.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package io.quarkiverse.operatorsdk.bundle.sources;
+
+import io.javaoperatorsdk.operator.api.reconciler.Context;
+import io.javaoperatorsdk.operator.api.reconciler.Reconciler;
+import io.javaoperatorsdk.operator.api.reconciler.UpdateControl;
+import io.quarkiverse.operatorsdk.bundle.runtime.CSVMetadata;
+
+@CSVMetadata(name = "first-operator")
```
Comment (metacosm) [inline bundle-generator/deployment/src/test/java/io/quarkiverse/operatorsdk/bundle/sources/FirstReconciler.java:8]: I think the default test should actually be to check that we generate a CSV per reconciler in the absence of `CSVMetadata` annotations.

**Review by metacosm — CHANGES_REQUESTED** [review]

File: bundle-generator/deployment/src/test/java/io/quarkiverse/operatorsdk/bundle/sources/FirstReconciler.java (hunk 1 above)
Comment (Sgitario) [reply bundle-generator/deployment/src/test/java/io/quarkiverse/operatorsdk/bundle/sources/FirstReconciler.java:8]: added

File: bundle-generator/deployment/src/test/java/io/quarkiverse/operatorsdk/bundle/sources/First.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import io.fabric8.kubernetes.api.model.Namespaced;
+import io.fabric8.kubernetes.client.CustomResource;
+import io.fabric8.kubernetes.model.annotation.Group;
+import io.fabric8.kubernetes.model.annotation.Version;
+
+@Group("samples.javaoperatorsdk.io")
+@Version("v1alpha1")
+@JsonInclude(JsonInclude.Include.NON_NULL)
```
Comment (metacosm) [inline bundle-generator/deployment/src/test/java/io/quarkiverse/operatorsdk/bundle/sources/First.java:12]: Why is this needed?

**Comment by Sgitario** [comment]
duplicated by https://github.com/quarkiverse/quarkus-operator-sdk/pull/312

File: bundle-generator/deployment/src/test/java/io/quarkiverse/operatorsdk/bundle/sources/First.java (hunk 2 above)
Comment (Sgitario) [reply bundle-generator/deployment/src/test/java/io/quarkiverse/operatorsdk/bundle/sources/First.java:12]: It's not needed. It's there because I copied it from the ping-pong example.
